include/revng/Pipeline/LLVMContainerFactory.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Pipeline/Context.h"
+#include "revng/Pipeline/LLVMContainer.h"
+
+namespace Pipeline {
+
+namespace detail {
+template<typename LLVMContainerType>
+ContainerFactory makeLLVMContainerFactoryImpl(Pipeline::Context *Ctx,
+                                              llvm::LLVMContext *Context) {
+  return [Ctx, Context](llvm::StringRef Name) {
+    auto Module = std::make_unique<llvm::Module>("rev.ng module", *Context);
```

Comment:
Why do we have something rev.ng-speific in here? Shall we use `Name` as the module name?